### DISCUSSION TITLE
Make Settings separate from Project Settings to modify defaults when creating new projects

### DISCUSSION THREAD

**Discussion by mysterycoconut** [godotengine/godot-proposals#9200] [Editor]
The defaults settings for Godot (the ones that every new project has as Project Settings) are part of the codebase, and not configurable. If there was a way to change them in a per machine basis (so they live in the appropriate Settings/Preferences folder of the operation system), it could ease a lot of the issues until we get project templates.

For example, I make mostly pixel games. Right now I have an empty project that I have to duplicate to get certain settings right (hidpi off, pixel perfect, etc…). If I could change all these defaults for the app, new projects could just read from there instead of the hardcoded Godot defaults.

This also has the advantage that it is how most other apps work. You install Word or Photoshop, it comes with its own defaults. You change them (sometimes with no documents open, sometimes through a different menu item), it writes that to your machine settings, and from now on all new documents use that. 

You could add this to the Project Manager window, as a "Settings" or "Default Settings" or "Defaults" button, and to the appropriate menu for the platform (for example, the Godot menu on a Mac).

**Comment by Calinou** [comment]
I feel a proper [project templates](https://github.com/godotengine/godot-proposals/issues/1481) solution is more suited to address the problem outlined here.

> hidpi off

You should **not** disable DPI awareness for pixel art games, as this is more likely to make them look blurry on hiDPI displays than anything else. Windows will use bilinear filtering for display scales that are not integer (such as 125% or 150%), and this will cause your game to look blurry (on top of not going fullscreen properly).

Every new game/application released in 2024 should be DPI-aware. Unfortunately, many modern games forget to do that on release and end up looking broken on hiDPI displays (e.g. with the aforementioned fullscreen issue).
